docs/internationalization/core.md

```diff
@@ -54,6 +55,42 @@ The translation strings for the configuration flow handler and the option flow h
 }
 ```
 
+### Selectors
+
+The translation for selectors are defined under the `selector` key. It supports option label translations for the selector `select`. The integration should set the `translation_key` on the selector select configuration. This allows translations on select selectors used in config and options flows. An example strings file below describes the different supported keys.
+
+```json
+{
+  "config": {
+    "flow_title": "Discovered Device ({host})",
+    "step": {
+      "init": {
+        "title": "The user visible title of the `init` step.",
+        "description": "Markdown that is shown with the step.",
+        "data": {
+          // Config flow selector select with options that support translations
+          "set_ca_cert": "Broker certificate validation"
+        }
+      }
+    }
+  },
+  // Translations for selector select to be used in option and config flows
+  "selector": {
+    // The key is linked to the `translation_key` that needs to be set
+    // together with `translation_domain` using the SelectSelectorConfig class
```

Comment:
Where is `translation_domain` explained?

Reply:
It has been removed, missed that one in my above change. Will PR it.

Reply:
https://github.com/home-assistant/developers.home-assistant/pull/1632 will remove the stale info.